Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM node:16-alpine3.11 AS build
```

Comment:
Should we use more generic image tags to always get the most recent base image (at least regarding the major version) at build time?